### ISSUE TITLE
Remove merge conflict for 21.7.0 release branch

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10234]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#10234|
|Type|enhancement|
|Created|2021-07-20 15:22:09Z|

The release branch has a merge conflict with trunk.

If a GitHub PR has a conflict, the pull_request jobs are not executed.

Under normal circumstances this is fixed by merging trunk into the PR branch...

but the PR branch is the release branch which should be merged only after the release.

<details><summary>Searchable metadata</summary>

```
trac-id__10234 10234
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__review review
time__1626794529327162 1626794529327162
changetime__1626796392783798 1626796392783798
version__None None
owner__Adi_Roiban__adiroiban_____ Adi Roiban <adiroiban@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

PR at https://github.com/twisted/twisted/pull/1635

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @adiroiban<br>@adiroiban set status to `closed`|
|-|-|

In changeset 9d62a9a57d01958f11fde7ffbfe3eadb9a40cb60
```
#!CommitTicketReference repository="" revision="9d62a9a57d01958f11fde7ffbfe3eadb9a40cb60"
Merge pull request #1635 from twisted/fix-21.7.0-release-pr

[10234] Delete file to prevent merge conflicts for the 21.7.0 PR.

Author: adiroiban
Reviewer: graingert
Fixes: ticket:10234
```
